test/serializers/default_include_test.rb

```diff
@@ -0,0 +1,52 @@
+require 'test_helper'
+
+module ActiveModel
+  class Serializer
+    class DefaultIncludeTest < ActiveSupport::TestCase
+      class Blog < ActiveModelSerializers::Model
+        attr_accessor :id, :name, :posts
+      end
+      class Post < ActiveModelSerializers::Model
+        attr_accessor :id, :title, :author
+      end
+      class Author < ActiveModelSerializers::Model
+        attr_accessor :id, :name
+      end
+      class BlogSerializer < ActiveModel::Serializer
+        attributes :id
+        attribute :name, key: :title
+
+        has_many :posts
+      end
+      class PostSerializer < ActiveModel::Serializer
+        default_include [:author]
+        attributes :id, :title
+
+        has_one :author
```

Comment:
but what does include all mean? include all dependencies? include all authors?

if anything, `include: :always` would make more sense.

Though, implementation is going to be slightly more complicated than the current.

> Advantage would include specifying included fields

can you explain thoughts on this?